php/functions.php

```diff
@@ -662,8 +662,8 @@ function getTeamEmails($db, $teamID=NULL, $tournamentID=NULL, $parents)
 			$output .= "<td><a href='mailto: ".$row['parent2Email']."'>".$row['parent2Email']."</a></td>";
 			$output .= "</tr>";
 
-			$emails[] = $row['email'];
-			$schoolEmails[] = $row['emailSchool'];
+			$emails[] = $row['parent1Email'];
+			$emails[] = $row['parent2Email'];
```

Comment:
This line overrides the parent1Email.

Reply:
Also, low level users should not have access to parent emails.  That is why it should be under userPrivilege(3).

Reply:
> This line overrides the parent1Email.

I thought using that syntax is the equivalent of appending each element to the end of the array like array_push(). I also tested it on a local version and it seemed to copy all emails correctly.

> Also, low level users should not have access to parent emails. That is why it should be under userPrivilege(3).

I believe I did put a check for userHasPrivilege(3) in the method. Does it need to be in a different place to prevent users from seeing the parent emails?